frontera/contrib/backends/hbase.py

```diff
@@ -22,6 +22,7 @@
 import six
 from six.moves import map
 from six.moves import range
+from w3lib.util import to_native_str
 
 
 _pack_functions = {
```

Comment:
I think that _pack_functions for URL and domain_fingerprint should be `to_bytes`. `str` returns native strings. I think we want happybase to receive bytes.